### DISCUSSION TITLE
[Request] Please change the upgrade scheme

### DISCUSSION THREAD

**Discussion by owlen** [music-assistant/support#2085] [Feature requests and ideas]
### What version of Music Assistant has the issue?

29.03.2024

### What version of the Home Assistant Integration have you got installed?

2024.3.1

### Have you tried everything in the Troubleshooting FAQ and reviewed the Open and Closed Issues and Discussions to resolve this yourself?

- [X] Yes

### The problem

I **really** appreciate all the effort that goes into making this great addon! I'm using the 2.0.0 BETA version.

However, since there are daily upgrades and updates, I keep getting daily notifications to update.
Again, every little update is appreciated, but since the "stable version" is 1, I wouldn't want to downgrade to it.

I think there can be a few very common alternatives, and I hope you can consider at least one of them:

1. Create an alfa version and update it frequently. This way, the beta can be updated less frequently, say, once a month.
2. Release a stable 2.0, or at least an RC for 2.0, and allow users to opt for that.
3. batch updates together at a lower frequency.

I'm sure many others and I would like to keep their HA installation up to date, and I don't like skipping updates. Many have scripts for alerts and notifications when the system has updates pending. 

Again, thanks a lot for the hard work, and I hope you can consider this.

### How to reproduce

Not applicable

### Music Providers

Not applicable

### Player Providers

Not applicable

### Full log output

Not applicable

### Additional information

Not applicable

### What version of Home Assistant Core are your running

2024.3.1

### What type of installation are you running?

Home Assistant OS

### On what type of hardware are you running?

Generic x86-64 (e.g. Intel NUC)

**Comment by OzGav** [comment]
Moved to a discussion. Marcel is working towards 2.0 stable

**Reply by owlen** [reply]
Thanks for the housekeeping and the update!

**Comment by marcelveldt** [comment]
Closed here as we all know we're working towards a 2.0 stable version. I find this request an insult even.

**Reply by owlen** [reply]
Of course, no insult was meant, and considering the tone and language of the OP, you could understand that.
Regardless, any project can only gain from listening to and even inviting criticism. Even if it is wrong.

Once again: "I **really** appreciate all the effort that goes into making this great addon" and I'm looking forward to the coming 2.0!

**Reply by marcelveldt** [reply]
The upgrade scheme will update as soon as 2.0 has been released, we're in between worlds now in a not so ideal scenario.

Just hold on and if the many updates bother you, its just because we want to squash the bugs that got introduced after some recent changes that had big impact. 

I am really not going to invest time in another update scheme in between. Uninstall or ignore your notifications in between and let us finish these last miles please.
